rosbag2/include/rosbag2/writer.hpp

```diff
@@ -95,13 +95,19 @@ class ROSBAG2_PUBLIC Writer
    */
   virtual void write(std::shared_ptr<SerializedBagMessage> message);
 
+  /**
+   * Check if the current recording database file needs to be split and rolled over to new file.
+   */
+  virtual bool should_split_database() const;
```

Comment:
Oh - not sure which PR is which - maybe this is the one where i ask "should we make this not be called 'database'" since that's a plugin detail?

Reply:
This was renamed to `should_split_bagfile` in PR #171 
I'll rebase this PR on top of that PR once we agree on the changes in PR #171.